www/src/public/js/controllers/builder/main.js

```diff
@@ -1311,9 +1338,10 @@
                 }
                 else if ($scope.selectedListIndex >= $scope.allLists.length) {
                     selectListByIndex($scope.allLists.length - 1);
+                    //$scope.allLists.length - 1
                 }
                 else {
-                    selectListByIndex($scope.selectedListIndex);
+                    selectListByIndex($scope.allLists.length - 1);
```

Comment:
What's the intended purpose for this change?

The old logic would make it so that if you delete a character and they're not at the end of your list of characters, it'll select the next character by default. This updated logic will make it so that it always selects the last character in your character list when you delete one

Reply:
I...have no idea.  I don't remember ever willfully making changes to the delete function.  I'll check the original and just move it back, no clue how that happened.